### DISCUSSION TITLE
Linqpad integration

### DISCUSSION THREAD

**Discussion by jeremylcarter** [DaveSkender/Stock.Indicators#1078] [Q&A]
Just wondering has anyone setup Linqpad to visualise indicators "quick and dirty"? As Linqpad supports System.Drawing you can `.Dump()` a graphics object and setup RX/Observables to redraw. I usually start tinkering in Linqpad before I go into VS.

See http://www.linqpad.net/customvisualizers.aspx

**Comment by DaveSkender** [comment]
Try asking in our [data visualization discussion thread](https://github.com/DaveSkender/Stock.Indicators/discussions/430) too.
